generators/app/templates/src/serviceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Is there a way to remove the need for this disable?

Reply:
This file is specific to CRA, therefore we probably should not touch it